broadinstitute/single_cell_portal_core
# Chunked file uploading and image file inclusion (SCP-3602)

SCP-3602 The UX for the image files will definitely change (and may be feature flagged depending on timing), but the technical pieces are straightforward.  

TO TEST:
1. check out branch, and turn on the 'react_upload_wizard' feature flag
2. go to upload wizard, confirm you can upload an image, associate it with a cluster, and then see it in the explore tab (note that the image preview currently requires to to refresh the page after upload -- that will get fixed in later polish)
3. Test uploading a file larger than the 10MB CHUNK SIZE (or go to UploadWizard.js and reduce CHUNK_SIZE to something smaller) and confirm it is correctly uploaded (and using the debugger network tab, confirm the chunk sizes are correct)
